packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -42,14 +42,30 @@ class ParameterMetadata(NamedTuple):
     Required when 'type' is Kind.TypeMessage. Ignored for any other 'type'.
     """
 
-    def sanitized_display_name(self) -> str:
-        """Parameter display name of alpha/numerical characters.
+    field_name: str = ""
+    """display_name in snake_case format."""
 
-        Returns:
-            str: Alpha/numerical characters of 'display_name'.
 
-        """
-        return "".join(char for char in self.display_name if char.isalnum())
+def initialize(
```

Comment:
[style] Make this a `@classmethod` inside the ParameterMetadata class so you can import ParameterMetadata and then call ParameterMetadata.initialize. I don't want it to be imported as just `initialize` like you did in the tests because that fails the "what if two people did this?" test and the resulting code is not clear what it's initializing.